app/jenkins-for-jira-ui/src/components/InProductHelpDrawer/InProductHelpAction.tsx

```diff
@@ -30,7 +44,7 @@ export const InProductHelpAction = ({
 		<>
 			<span
 				role={actionRole}
-				className={cx(inProductHelpTypeClassName)}
+				className={cx(inProductHelpTypeClassName, inProductHelpButtonStyles)}
```

Comment:
Realised that Atlaskit button has a hover and active state that I neglected to add when I made the switch to ensure our UI matched the designs.